packages/toolpad-app/src/server/appTemplateDoms/doms.ts

```diff
@@ -1,11 +1,19 @@
+import * as path from 'path';
 import * as appDom from '../../appDom';
 
-import statisticsAppDom from './statistics.json';
-import imagesAppDom from './images.json';
-import { AppTemplateName } from '../../types';
+import { AppTemplateId } from '../../types';
+import { readJsonFile } from '../../utils/fs';
 
-export const APP_TEMPLATE_DOMS: Record<AppTemplateName, appDom.AppDom | null> = {
+// __dirname does not work, see: https://github.com/vercel/next.js/discussions/14341
+const DOMS_DIR_PATH = './src/server/appTemplateDoms';
```

Comment:
not ideal but i could not find a way to use relative paths as `__dirname` resolves to `pages/api`
i've added a test to ensure selecting templates works as expected